### PR TITLE
Qusetions 

### PR DESCRIPTION
**Pull request by AliOthman0934**

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -56,6 +56,182 @@ export const quizData = {
...
+        {
+          text: 'MDN',
+          href:
+            'https://developer.mozilla.org/en-US/docs/Web/CSS',
+        },
+      ],
+    },
     // Add more questions here
   ],
 };
```
Comment (mariamisaeva) [inline src/data.js:237]: Done 👍

**Comment by burhan1997** [comment]
Super

**Comment by mariamisaeva** [comment]
Nice

File: src/data.js [hunk 2]
```diff
@@ -56,6 +56,182 @@ export const quizData = {
         },
       ],
     },
+    {
+      text: 'Which HTML attribute is used to define inline styles?',
+      answers: {
+        a: 'font',
+        b: 'class',
+        c: 'style',
+        d: 'css',
```
Comment (burhan1997) [inline src/data.js:65]: Nice

**Review by burhan1997 — APPROVED** [review]

**Comment by burhan1997** [comment]
Very nice feature

**Review by mariamisaeva — APPROVED** [review]

**Pull request merged**